pkgs/development/libraries/vapoursynth/default.nix

```diff
@@ -37,12 +37,22 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  outputs = [ "out" "dev" "python" ];
+
+  postInstall = ''
+    moveToOutput include $dev
+    moveToOutput lib/${python3.libPrefix} $python
+    moveToOutput bin/vspipe $python
+    wrapProgram $python/bin/vspipe \
+        --prefix PYTHONPATH : $(toPythonPath $python)
```

Comment:
Use `wrapPythonPrograms` from `python3.pkgs.wrapPython`, instead.

Reply:
This doesn't seem to work with binary programs.

Reply:
It does work with binary programs (it iterally calls `wrapProgram` under the hood), but it relies on `$pythonPath` being set in the builder. This happens automatically for `buildPythonPackage`, but not for `stdenv.mkDerivation`.

You could either set `pythonPath` appropriately, or actually use `buildPythonPackage`, which would possibly be appropriate given it provides a Python library. I **suspect** if you do that and specify `format = other;`, it may Just Work (tm) and you can drop the manual wrapping entirely, but I haven't tested it :man_shrugging:.